bin/roave-infection-static-analysis-plugin

```diff
@@ -56,10 +57,61 @@ use function var_export;
         define('PHP_PARSER_VERSION', Versions::getVersion('nikic/php-parser'));
     }
 
-    $makeAnalyzer = static function () use ($projectPath): ProjectAnalyzer {
+    $extractArgument =
```

Comment:
Can we somehow abstract this into an `@internal` (tested) utility? 😁

Reply:
Yep, will do.